pandas/core/base.py

```diff
@@ -7,8 +7,10 @@
 from pandas.core import common as com
 import pandas.core.nanops as nanops
 import pandas.lib as lib
+from pandas.compat.func_defaults import TRANSPOSE_DEFAULTS
```

Comment:
its the same idea as with ``validate_args_and_kwargs``, but needs special handling per-function, so need to put it somewhere. its just cleaner to put it hidden away.